Sources/SolToBoogie/ProcedureTranslator.cs

```diff
@@ -1632,6 +1639,9 @@ private void TranslateAbiEncodedFuncCall(FunctionCall funcCall, BoogieExpr lhs)
             }
             var boogieExprs = arguments.ConvertAll(x => TranslateExpr(x));
             var funcName = $"abiEncodePacked{arguments.Count}";
+            // hack

```

Comment:
Oooh! What is this for?

Reply:
To determine the function name according to the type of the first argument. Not supper crafted design. This is why I said "hack". I need to have this encodings for the Zeppelin case study.

Reply:
I have moved this commit to another PR